src/config/dynamic-theme-fixes.config

```diff
@@ -1029,6 +1029,12 @@ CSS
 .InboxExpandableThread.InboxExpandableThread--selected {
     background: var(--darkreader-neutral-background) !important;
 }
+.PortfolioItemRow {
+    background: var(--darkreader-neutral-background) !important;
+}
+.PortfolioItemRow:hover {
```

Comment:
```CSS
var(--darkreader-neutral-hover)
```
This isn't a variable.

Reply:
Hmm. If I remove this line, the hover-highlighting feature stops working, so it must be doing something. Any ideas?

Reply:
I don't have a clue, it should be ignored CSS as it's invalid variable. So I suggest setting it to something like `${#333}` or anything else that's appropriate.